src/transformers/integrations/bitsandbytes.py

```diff
@@ -243,6 +243,10 @@ def replace_with_bnb_linear(model, modules_to_not_convert=None, current_key_name
             An array to track the current key of the recursion. This is used to check whether the current key (part of
             it) is not in the list of modules to not convert (for instances modules that are offloaded to `cpu` or
             `disk`).
+        quantization_config ('transformers.utils.quantization_config.BitsAndBytesConfig'):
```

Comment:
```suggestion
        quantization_config ('~BitsAndBytesConfig'):
```
This should work in our docs